### PR TITLE
Test OffscreenCanvas.transferToImageBitmap compositing steps (#3612)

### PR DESCRIPTION
**Pull request by kkinnunen-apple**
Test following spec clarifications:
  * the value of preserveDrawingBuffer will have no effect
  * the contents of any depth and/or stencil buffers will be cleared to initial values

### CODE REVIEW

File: sdk/tests/js/tests/offscreencanvas-transfer-image-bitmap.js [hunk 1]
```diff
@@ -1,3 +1,181 @@
...
+    let fs = gl.createShader(gl.FRAGMENT_SHADER);
+    gl.shaderSource(fs, "precision mediump float; void main() { gl_FragColor = vec4(1., 1., 1., 1.); }");
+    gl.compileShader(fs);
+    let p = gl.createProgram();
+    gl.attachShader(p, vs);
+    gl.attachShader(p, fs);
+    gl.bindAttribLocation(p, 0, "a_pos");
+    gl.linkProgram(p);
+    gl.useProgram(p);
+    gl.viewport(0, 0, 128, 128);
```
Comment (kdashg) [inline sdk/tests/js/tests/offscreencanvas-transfer-image-bitmap.js:15]:
```suggestion
    gl.viewport(0, 0, gl.canvas.width, gl.canvas.height);
```

Comment (kdashg) [reply sdk/tests/js/tests/offscreencanvas-transfer-image-bitmap.js:15]: It's nice to hardcode fewer things.

**Review by kdashg — APPROVED** [review]

**Review by kenrussell — APPROVED** [review]
Sorry for dropping this - looks good as long as it's testing the agreed-upon spec.

**Comment by kenrussell** [comment]
Shall we merge this? It was promised to merge it at the last WebGL working group conference call ~2 weeks ago.

**Comment by kenrussell** [comment]
Merging per discussion in WebGL WG.

**Pull request merged**